libs/dal/src/repositories/message-template/message-template.schema.ts

```diff
@@ -82,11 +82,6 @@ const messageTemplateSchema = new Schema<MessageTemplateDBModel>(
     controls: { schema: Schema.Types.Mixed, uiSchema: Schema.Types.Mixed },
     output: { schema: Schema.Types.Mixed },
     code: Schema.Types.String,
-    severity: {
-      type: Schema.Types.String,
-      enum: SeverityLevelEnum,
-      default: SeverityLevelEnum.NONE,
-    },
```

Comment:
this was done by mistake, I confused `MessageEntity` with `MessageTemplateEntity` 🤦